src/main/java/lotto/console/domain/user/CreationCount.java

```diff
@@ -0,0 +1,47 @@
+package lotto.console.domain.user;
+
+import java.util.Objects;
+
+public class CreationCount {
+
+	private static final int TOTAL_COUNT_PER = 1000;
+
+	private final int totalCount;
+	private final int automaticCount;
+	private final int manualCount;
+
+	public CreationCount(final Money money, final int manualCount) {
+		Objects.requireNonNull(money);
+		this.totalCount = money.getMoney() / TOTAL_COUNT_PER;
+		validateNegativeManualCount(manualCount);
+		validateGreaterManualCountThanTotalCount(this.totalCount, manualCount);
+		this.manualCount = manualCount;
+		this.automaticCount = totalCount - manualCount;
+	}
+
+	private void validateGreaterManualCountThanTotalCount(int totalCount, int manualCount) {
```

Comment:
```suggestion
	private void validateManualCountAndTotalCount(int totalCount, int manualCount) {
```
로도 충분하지 않을까요...? 결과적으로는 두 값의 유효성을 확인하는 메소드니까요!

Reply:
명확하게 명시하고 싶었습니다. 이 유효성 검사는 총 횟수보다 수동 횟수가 더 큰 경우라서요.. ㅎ